schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+              headers:
+                type: object
+                additionalProperties:
+                  $ref: '#Header'
+        unevaluatedProperties: false
+      Example:
+        $anchor: Example
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            properties:
+              summary:
+                type: string
+              value: true
+              externalValue:
+                type: string
+                format: uri-reference
+            unevaluatedProperties: false
+      Header:
+        $anchor: Header
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            required:
+              - schema
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/DescReqDep'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - allOf:
+                  - $ref: '#/$defs/Mixins/$defs/WithSchemaAndExamples'
+                  - $ref: '#/$defs/Mixins/$defs/StyledSimple'
+              - allOf:
+                  - $ref: '#/$defs/Mixins/$defs/WithContent'
+                  - $ref: '#/$defs/Mixins/$defs/WithEmpty'
+            unevaluatedProperties: false
+      Paths:
+        $anchor: Paths
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        patternProperties:
+          ^/:
+            type: object
+            oneOf:
+              - $ref: '#Reference'
+              - $ref: '#PathItem'
+        unevaluatedProperties: false
+      PathItem:
+        $anchor: PathItem
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          summary:
+            type: string
+          servers:
+            type: array
+            items:
+              $ref: '#Server'
+          parameters:
+            type: array
+            items:
+              oneOf:
+                - $ref: '#Parameter'
+                - $ref: '#Reference'
+            uniqueItems: true
+      Operation:
+        $anchor: Operation
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Deprecatable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          tags:
+            type: array
+            items:
+              type: string
+          summary:
+            type: string
+          externalDocs:
+            $ref: '#ExternalDocumentation'
+          operationId:
+            type: string
+          parameters:
+            type: array
+            items:
+              $ref: '#Parameter'
+          requestBody:
+            $ref: '#RequestBody'
+          responses:
+            $ref: '#Responses'
+          callback:
+            $ref: '#Callback'
```

```suggestion
          callbacks:
            type: object
            additionalProperties:
              $ref: '#Callback'
```
See [`callbacks`](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#user-content-operationcallbacks)